release-notes/release-notes53.md

```diff
@@ -47,6 +47,16 @@ For example:
 
 For information on toggle menu items, see: [Custom menu items - Toggle menu items]({{site.baseurl}}/ui-components/menuitems/#togglemenuitems).
 
+### New find and replace in selection functionality
+
+A new "Find in selection" option has been added to the Find and Replace dialog provided by the Search and Replace plugin.
+
+For example:
+
+![Find and Replace dialog with new "Find in selection" option highlighted]({{site.baseurl}}/images/findReplaceInSelection.png)
```

Comment:
Hmm... this image name seems very inconsistent with the rest in the docs. It normally uses hyphens instead of camel case.

Reply:
fixed by force-push